package.json

```diff
@@ -453,16 +453,20 @@
 		]
 	},
 	"scripts": {
-		"dev": "webpack --config webpack.config.preview.js --mode development && npm run compile",
-		"compile": "tsc -p ./",
-		"package": "webpack --config webpack.config.preview.js --mode production && npm run compile && vsce package",
+		"copy-assets": "cp node_modules/@highlightjs/cdn-assets/highlight.min.js media/highlightjs && cp -r node_modules/@highlightjs/cdn-assets/languages media/highlightjs && cp -r node_modules/@highlightjs/cdn-assets/styles media/highlightjs",
+		"dev": "npm run build",
+		"build": "npm run copy-assets && npm run build-ext && npm run build-preview",
+		"build-preview": "webpack --config webpack.config.preview.js --mode production",
+		"build-ext": "tsc -p ./",
```

Comment:
I took inspiration from https://github.com/microsoft/vscode/blob/a4342f3addc29f405a5cb4aab359b0b98223411e/extensions/markdown-language-features/package.json#L344-L352 and renamed the tasks to "build" (instead of "compile").

As a result, I included the "copy assets" task in the "build" task.

We are now using `--mode production` even when running the dev or test task. That's what the Markdown extension is doing. It simplify things and, as far as I know, it has no impact since we can still debug the extension using VS code... right?